client/blocks/inline-help/popover.jsx

```diff
@@ -205,17 +235,51 @@ class InlineHelpPopover extends Component {
 		);
 	};
 
+	checkForClassicEditorOnAtomic() {
+		const { siteId, sitePlugins, showErrorNotice, translate } = this.props;
+		const classicPlugin = find( sitePlugins, { slug: 'classic-editor' } );
+
+		if ( ! classicPlugin ) {
+			showErrorNotice(
+				translate(
+					'There was a problem activating the Classic editor on your site. Please contact support.'
```

Comment:
I'm a bit afraid of that "please contact support". 😄 

What if we suggested to go to the plugins page and activating Classic manually instead?
I don't have a clear flow in mind so I'm just throwing a random idea here, but maybe even just something like "please go to the plugins page, activate Classic Editor, and come back here" might help alleviating the HE load.

Reply:
Good point, I changed it to 'There was a problem activating the Classic editor on your site. Please go to the plugins page and activate the Classic Editor plugin there.'